coalib/misc/Constants.py

```diff
@@ -13,6 +13,11 @@
                  "happen. When asked for, the following information "
                  "may help investigating:")
 
+VERSION_CONFLICT_MESSAGE = ("There is a conflict in the version of a package "
+                            "you have installed and the requirements of "
+                            "coala. The following information may help "
+                            "to resolve this conflict: ")
```

Comment:
Ah, I think we can access the requirement package (with correct version). I wonder if `pip3 install <package_name>` would work for the string returned by pkg_resources.